### ISSUE TITLE
Swipe Gesture for Main Menu

### ISSUE THREAD

**Issue by vrindavan** [TeamNewPipe/NewPipe#5399]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want
The left side menu needs to be opened by touching the 3 lines hamburger manu which is very ridiculous for users who have bigger screens. I am requesting a swipe gesture, ability for user to just swipe right from the screen's main page and the menu will open.

**Comment by mhmdanas** [comment]
Well, actually, you can swipe anywhere from the left edge of the screen and the hamburger menu will open.

**Comment by vrindavan** [comment]
Notnworking.

**Comment by mhmdanas** [comment]
Weird. I just tested it and it works. Mind posting a screen recording?

**Comment by vrindavan** [comment]
[VIDEO](https://drive.google.com/file/d/11QTOnaXUHj7LvHyWDt4PRfFBAkUu38Th/view?usp=drivesdk)

Once seen let me know, I'll delete it.

**Comment by mhmdanas** [comment]
Yes, I saw it. You're not swiping from the edge of the screen; you're swiping slightly right of it. I'll post a screenshot now to show how you can do it.

**Comment by vrindavan** [comment]
No, that is the problem. These are full screen gestures so I cannot do that otherwise it will press back.

**Comment by vrindavan** [comment]
NewPipe needs to have swipe enabled from the main screen for it to work on the full screen gesture devices.

**Comment by mhmdanas** [comment]
Ah, makes sense. That piece of context is very important, so you probably want to edit the main issue to include it. Unfortunately, I don't think swiping from amywhere to open the menu will be added since it is also used to move between the tabs. However, I'll re-open this issue just in case.

**Comment by vrindavan** [comment]
Oh, sure I will edit this.

**Comment by vrindavan** [comment]
Done, check now @mhmdanas

**Comment by HTSamurai** [comment]
> No, that is the problem. These are full screen gestures so I cannot do that otherwise it will press back.

My phone is fullscreen gestures but I can do it. Just swipe in the upper half of the screen

**Comment by HTSamurai** [comment]
Look 
![Screenshot_2021-01-13-11-44-07-209_org schabi newpipe](https://user-images.githubusercontent.com/71007160/104448300-e1fe6b00-5594-11eb-940f-7944acc6b6fe.jpg)
![Screenshot_2021-01-13-11-44-18-827_org schabi newpipe](https://user-images.githubusercontent.com/71007160/104448311-e62a8880-5594-11eb-8366-093284618412.jpg)

**Comment by vrindavan** [comment]
@HTSamurai i cannot do it still :/

**Comment by HerrHulaHoop** [comment]
I'm on OnePlus and face the same issue. The back button covers the entire sides of the screen. 

Would be neat if an option is added to move the contents of the sidebar into a 4th tab in the homescreen. Then we could simply swipe to it if we need to reach it. Bit of a stretch but if there was a way to auto-detect the gestures and enable this, it would be even better.

**Comment by Frivolous2** [comment]
Google has an article describing how to get around this problem, e.g. to make the app opt out of the "back" gesture in certain views:

https://developer.android.com/develop/ui/views/touch-and-input/gestures/gesturenav
